examples/plot_linestrengths.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+================================================
+Scale Linestrengths of carbon-monoxide
+================================================
+
+This example scales the linestrengths of CO to Tgas=300 from Tref = 296 and then plots
+the linestrengths against the wavenumbers. We are using :py:func:`~radis.io.hitemp.fetch_hitemp`
+function to retrieve the dataframe from the HITEMP-CO databank. The calculations done
+for scaling use the  equation that can be found in Rotham's paper (equation A11).
+
+"""
```

Comment:
We can use these examples to give information about what's done in the code (good for future spectroscopy tutorials !)

```suggestion
Similar functions are used directly at the hearth of RADIS's SpectrumFactory in the :py:meth:`~radis.lbl.BaseFactory.calc_linestrength_eq` and :py:meth:`~radis.lbl.BaseFactory.calc_linestrength_noneq` methods
"""
```

I also suggest that you : 
- borrow the equation and link to the references of BaseFactory.calc_linestrength_eq docstrings, and add it here. Will look very nice !
- mention this example in BaseFactory.calc_linestrength_eq docstring, by adding 
```
.. minigallery:: radis.lbl.base.BaseFactory.calc_linestrength_eq
```
- make BaseFactory._calc_linestrength_noneq a public method by renaming it BaseFactory.calc_linestrength_noneq